textworld/generator/text_grammar.py

```diff
@@ -117,16 +133,16 @@ def __init__(self, options: Union[GrammarOptions, Mapping] = {}, rng: Optional[R
 
         # Load the grammar associated to the provided theme.
         self.theme = self.options.theme
-        grammar_contents = []
 
         # Load the object names file
         files = os.listdir(KnowledgeBase.default().text_grammars_path)
         files = [f for f in files if f.startswith(self.theme + "_") and f.endswith(".twg")]
```

Comment:
While at it, I think we should do 
```suggestion
        files = glob(pjoin(KnowledgeBase.default().text_grammars_path, self.theme + "_*.twg')
```